src/controllers/order.controller.js

```diff
@@ -1,77 +1,4 @@
-import WebSocket from 'ws';
-import { orderServices, userServices } from '../services';
-
-// // Initialising web sockets
-// const wss = new WebSocket.Server({ port: 4000 });
```

Comment:
can you remove these commented ones?incase they aren't needed?

Reply:
They are removed, that's why they in red